### PR TITLE
Adds help button back

### LINKED ISSUE #3394: Paper help button is missing
Where'd it go?

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

obraz

Adds help button back
Save and Confirm are capitalized again

closes: #3394

## Why It's Good For The Game

ike asked to

## Changelog
:cl:
add: Help button is back on paper.
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/PaperSheet.js [hunk 1]
```diff
@@ -564,11 +577,178 @@ class PaperSheetEdit extends Component {
...
+          </Table.Row>
+        </Table>
+      </Box>
+    );
+  }
+
 }
 
+
+
```
Comment (Archanial) [inline tgui/packages/tgui/interfaces/PaperSheet.js]:
```suggestion
```

**Comment by ike709** [comment]
Post a pic of the actual help page. Also doesn't look like you included signatures or fields.

**Comment by Archanial** [comment]
obraz

**Comment by yyzsong** [comment]
The position of the markdowns and their examples switch halfway through...

**Comment by Archanial** [comment]
obraz

**Pull request merged**